cmd/benchmark/main.go

```diff
@@ -204,20 +216,27 @@ func main() {
 		CfgSize: options.NumNodes,
 	}
 
-	cfg, err := mgr.NewConfiguration(qspec, gorums.WithNodeList(remotes[:options.NumNodes]))
-	checkf("Failed to create configuration: %v", err)
+	cfg, err := mgr.NewConfiguration(qspec, gorums.WithNodeList(options.Remotes[:options.NumNodes]))
+	err = checkf("Failed to create configuration: %v", err)
```

Comment:
We should just drop the `checkf` thing and do:
```go
cfg, err := ...
if err != nil {
   return fmt.Errorf("Failed to create configuration: %w", err)
}
```
Or, alternatively just return the original error directly:
```go
if err != nil {
   return err
}
```
And similar for all the uses of `checkf` below in the `runClient` function. That is, we should only use `checkf` when we don't want to return an error but just exit if `err != nil`.